projects/plugins/jetpack/to-test.md

```diff
@@ -1,64 +1,31 @@
-## Jetpack 10.11
+## Jetpack 11.1
 
 ### Before you start:
 
 - **At any point during your testing, remember to [check your browser's JavaScript console](https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/#step-3-diagnosis) and see if there are any errors reported by Jetpack there.**
 - Use the "Debug Bar" or "Query Monitor" WordPress plugins to help make PHP notices and warnings more noticeable and report anything of note you see.
 
-### Jetpack connection sharing
+### Lazy Images [[#24643]](https://github.com/Automattic/jetpack/pull/24643)
 
-We improved how Jetpack shares the connection with Jetpack standalone plugins. To test this, try the following:
+We've updated Jetpack's [Lazy Images feature](https://jetpack.com/support/lazy-images/) to be compatible with a recent Safari browser update which enables native lazy loading of images by default. To test, try the following:
 
-- Connect Jetpack.
-- Activate a secondary plugin like Boost and/or Backup.
-- Go to Plugins.
-- Confirm the link to deactivate Jetpack says only "Deactivate" (and not 'Disconnect and Deactivate').
-- Click Deactivate.
-- Confirm the dialog shows information about the active features you have and no mention to other plugins.
-- Reconnect Jetpack and try disabling standalone plugins.
-- Make sure disconnecting from the Jetpack Dashboard still works.
+- Make sure you are using Safari version 15.4 or later to test with.
```

Comment:
Should we mention that this is the version you'll now find on iOS if you run iOS 15.4+? Ideally, I think that's where we would want folks to test.

```suggestion
- Make sure you are using Safari version 15.4 or later to test with. If you have an iPhone or iPad running iOS 15.4+, this will be the perfect place to test that change.
```